docker-compose.yml

```diff
@@ -7,6 +7,9 @@ services:
         ports:
             - '${HTTP_PORT}:80'
             - '${HTTPS_PORT}:443'
+        expose:
```

Comment:
Why is this needed?

Reply:
I tested it in two cases:
1. the container on the same host with prosody and web (I guess its not good idea)
2. the container on a different host
So, in the first case I can use it through internal docker bridge (like meet.jitsi) and I exposed the ports in the web container. For the second case it's not needed.

Reply:
So, as a starting point, we are only interested in case 1. In the future, this should be solved by something like Docker Swarm, that is, by having multi-host transparent uder defined networking (the meet.jitsi network).

In this case, I don't thing this is necessary.